mlrun/model_monitoring/db/stores/sqldb/sql_store.py

```diff
@@ -333,6 +562,26 @@ def _validate_labels(
 
         return True
 
+    @staticmethod
+    def filter_endpoint_and_application_name(
+        endpoint_id: str, application_name: str
+    ) -> dict[str, str]:
+        """Generate a dictionary filter for endpoint id and application name"""
+        if not endpoint_id and not application_name:
+            raise mlrun.errors.MLRunNotFoundError(
+                "Please provide a valid endpoint_id and/or application_name"
+            )
```

Comment:
```suggestion
        if not endpoint_id and not application_name:
            raise ValueError(
                "Please provide a valid endpoint_id and/or application_name"
            )
```

Reply:
instead of `ValueError` I will use `mlrun.errors.MLRunBadRequestError` because it's indeed a bad request error...